src/AutoRest.CSharp.V3/Properties/launchSettings.json

```diff
@@ -95,6 +95,10 @@
     "body-datetime": {
       "commandName": "Project",
       "commandLineArgs": "--standalone --input-codemodel=$(SolutionDir)\\test\\TestServerProjects\\body-datetime\\CodeModel.yaml --plugin=cs-modeler --output-path=$(SolutionDir)\\test\\TestServerProjects\\body-datetime --namespace=body_datetime"
+    },
```

Comment:
You've also generated `custom-baseUrl-paging` can you update launchSettings to include it?

Reply:
Done.